README.md

```diff
@@ -3,17 +3,24 @@ appium-chromedriver
 
 [![Release](https://github.com/appium/appium-chromedriver/actions/workflows/publish.js.yml/badge.svg)](https://github.com/appium/appium-chromedriver/actions/workflows/publish.js.yml)
 
-Node.js wrapper around [Chromedriver](https://sites.google.com/a/chromium.org/chromedriver/).
+Node.js wrapper around [Chromedriver](https://sites.google.com/chromium.org/driver/)
+and [Microsoft Edge WebDriver](https://developer.microsoft.com/en-us/microsoft-edge/tools/webdriver/).
```

Comment:
we should also mention what is the minimum module version to support the edge driver